alembic/testing/plugin/pytestplugin.py

```diff
@@ -50,6 +52,30 @@ def pytest_pycollect_makeitem(collector, name, obj):
         return []
 
 
+def vendored_exclusions(compound_object):
```

Comment:
As you mentioned, the testing construct isn't in sqlalchemy before 1.3. So I did this janky thing- though I think it may make more sense to just drop the support for older versions as you mentioned.